pom.xml

```diff
@@ -54,7 +54,11 @@
       <!-- Sonar - START -->
       <sonar.github.repository>eclipse/hawkbit-extensions</sonar.github.repository>
       <sonar.links.ci>https://circleci.com/gh/eclipse/hawkbit-extensions</sonar.links.ci>
-      <jacoco.outputDir>${project.basedir}/../../target/</jacoco.outputDir>
+
+       <!-- Override Sonar/Reporting settings -->
+       <sonar.coverage.jacoco.xmlReportPaths>
+           ${project.basedir}/target/site/jacoco/jacoco.xml, ${project.basedir}/target/site/jacoco-it/jacoco.xml
```

Comment:
Better use `project.build.directory` instead of `project.basedir`. This will remove the need to add the `target` folder to the path. So this should be sufficient:

```
${project.build.directory}/site/jacoco/jacoco.xml, ${project.build.directory}/site/jacoco-it/jacoco.xml
```

Reply:
Done.